warriorsOfTheDarkGods.cat

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<catalogue id="9cd6-2c84-c59e-7dd1" name="Warriors of the Dark Gods 2.2.4" revision="47" battleScribeVersion="2.02" authorName="Mikel2311, DarkSky" library="false" gameSystemId="aa64-1e8e-66fc-9abf" gameSystemRevision="2" xmlns="http://www.battlescribe.net/schema/catalogueSchema">
+<catalogue id="9cd6-2c84-c59e-7dd1" name="Warriors of the Dark Gods 2.1.1" revision="48" battleScribeVersion="2.02" authorName="Mikel2311, DarkSky" library="false" gameSystemId="aa64-1e8e-66fc-9abf" gameSystemRevision="2" xmlns="http://www.battlescribe.net/schema/catalogueSchema">
```

Comment:
According to the file name, this is version 2.3 (seems to be an error in the document itself, as 2.1.1 is actually much older)

Reply:
I noticed that too during the changes but have now brought it up with the RT for clarification. 

Whatever they decide though, I believe that the BS files should correspond directly with the PDF they're trying to convey. The current WdG PDF says 2.1.1 and the previous was 2.2.x so the version decrease is originating in the source documents and not a typo from my side.

I suggest that the PDF and BS data files are kept in sync regardless. If this is a mistake, then the RT will release new PDF with a new version and we'll update the BS data file based on that.